### ISSUE TITLE
Can't run WFC or Zentangle

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#953]
I get this exception from a Zentangle run:
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/xml/serializer/OutputPropertiesFactory
```
This is in the branch where I am deleting a lot of POM/Maven stuff, so I should see if it is a general problem or just in that branch.

**Comment by schrum2** [comment]
The problem seems to be the xalan library, where the org.apache.xml.serializer package is totally empty. I looked at the original WFC code and it doesn't use this library at all. It looks like this code was added by Anna or Sarah to create adjacency rules for WFC to read, but the particular library does not work any more.

I might need to re-write the code in the writeAdjacencyRules method of SimpleTiledZentangleWFCModel. A comment above the method says the inspiration for the XML approach comes from:
https://crunchify.com/java-simple-way-to-write-xml-dom-file-in-java/
